tomster-player/app/styles/app.scss

```diff
@@ -61,6 +61,26 @@ $playerHeight: 75px;
   }
 }
 
+$sidebarWidth: 200px;
+
+.sidebar {
+  float: left;
+  width: $sidebarWidth;
+
+  .album-cover {
+    width: 100%;
+  }
+
+  .rating, .song-list {
+    display: none;
```

Comment:
Are you sure? There's no point because now the `album-list` is only rendered as a sidebar.